### PR TITLE
SBERDOMA-583 fix error with deletedAt and boolean access on read

### PR DESCRIPTION
**Pull request by toplenboren**
`deletedAt_not` null does not work thorough `GraphiQL` or python code. This PR fixes it

### CODE REVIEW

File: apps/condo/domains/billing/access/BillingIntegration.js [hunk 1]
```diff
@@ -5,7 +5,8 @@
 const { throwAuthenticationError } = require('@condo/domains/common/utils/apolloErrorFormatter')
 
 async function canReadBillingIntegrations ({ authentication: { item: user } }) {
-    return !!user
+    if (!user) return false
```
Comment (pahaz) [inline apps/condo/domains/billing/access/BillingIntegration.js:8]: Throw exception

File: apps/condo/domains/billing/access/BillingMeterResource.js [hunk 2]
```diff
@@ -8,7 +8,8 @@ const { checkBillingIntegrationAccessRight } = require('@condo/domains/billing/u
 const { throwAuthenticationError } = require('@condo/domains/common/utils/apolloErrorFormatter')
 
 async function canReadBillingMeterResources ({ authentication: { item: user } }) {
-    return !!user
```
Comment (pahaz) [inline apps/condo/domains/billing/access/BillingMeterResource.js:11]: Throw exception

File: packages/@core.keystone/plugins/softDeleted.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
 const { DateTimeUtc } = require('@keystonejs/fields')
 const { getType } = require('@keystonejs/utils')
 const { get, isPlainObject } = require('lodash')
-
+const { Parser } = require('graphql')
```
Comment (Dimitreee) [inline packages/@core.keystone/plugins/softDeleted.js:4]: unused import

File: apps/condo/domains/billing/schema/BillingIntegration.test.js [hunk 4]
```diff
@@ -1,3 +1,5 @@
+import { expectToThrowAuthenticationErrorToObjects } from '../../common/utils/testSchema'
```
Comment (nomerdvadcatpyat) [inline apps/condo/domains/billing/schema/BillingIntegration.test.js:1]: use domain import

File: apps/condo/domains/billing/schema/BillingIntegration.test.js [hunk 5]
```diff
@@ -6,7 +8,7 @@ const { getRandomString } = require('@core/keystone/test.utils')
 const { makeClientWithNewRegisteredAndLoggedInUser, makeClientWithSupportUser } = require('@condo/domains/user/utils/testSchema')
 const { makeLoggedInAdminClient, makeClient } = require('@core/keystone/test.utils')
 const { BillingIntegration, createTestBillingIntegration, updateTestBillingIntegration } = require('@condo/domains/billing/utils/testSchema')
-const { expectToThrowAccessDeniedErrorToObjects, expectToThrowAuthenticationErrorToObjects, expectToThrowAccessDeniedErrorToObj, expectToThrowAuthenticationErrorToObj } = require('../../common/utils/testSchema')
+const { expectToThrowAccessDeniedErrorToObj, expectToThrowAuthenticationErrorToObj } = require('../../common/utils/testSchema')
```
Comment (nomerdvadcatpyat) [inline apps/condo/domains/billing/schema/BillingIntegration.test.js:11]: @condo/domains

**Review by Dimitreee — APPROVED** [review]

**Review by nomerdvadcatpyat — APPROVED** [review]

**Pull request merged**